mathics/doc/common_doc.py

```diff
@@ -138,6 +138,28 @@
 test_result_map = {}
 
 
+# TODO: Use this when checking requires for modules.
+requires_lib_cache = {}
+
+
+def check_requires_list(requires: list) -> bool:
+    global requires_cache
```

Comment:
This line does nothing.

Reply:
I put this for clarity, but OK.

Reply:
Huh? There is no variable `requires_cache`.  The typical way of adding clarity is in a docstring. For example:

```
"""
Check if module names  in ``requires`` can be imported and return True if they can or False if not.

This state value is also recorded in dictionary `requires_lib_cache` keyed by package name and is used to determine whether to skip trying to get information from the package.
"""
```

Reply:
You are right! I changed the name. The variable was `requires_lib_cache`, which was defined just above.

Reply:
Since ```mathics.builtin.base``` also benefits from using ```check_requires_list``` I moved the code there.